datafusion/core/src/datasource/physical_plan/parquet/statistics.rs

```diff
@@ -190,20 +190,29 @@ pub(crate) fn parquet_column<'a>(
     name: &str,
 ) -> Option<(usize, &'a FieldRef)> {
     let (root_idx, field) = arrow_schema.fields.find(name)?;
-    if field.data_type().is_nested() {
-        // Nested fields are not supported and require non-trivial logic
-        // to correctly walk the parquet schema accounting for the
-        // logical type rules - <https://github.com/apache/parquet-format/blob/master/LogicalTypes.md>
-        //
-        // For example a ListArray could correspond to anything from 1 to 3 levels
-        // in the parquet schema
-        return None;
-    }
+    match field.data_type() {
+        DataType::Struct(_) => {
+            let parquet_idx = (0..parquet_schema.columns().len())
```

Comment:
@tustvold  could you help here -- is this the correct way to find the correct parquet leaf from an arrow struct type?